cmd/training-operator.v1/main.go

```diff
@@ -131,7 +134,11 @@ func main() {
 		}
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	cfg := ctrl.GetConfigOrDie()
+	cfg.QPS = float32(clientQps)
+	cfg.Burst = clientBurst
```

Comment:
```suggestion
	cfg.RateLimiter = flowcontrol.NewTokenBucketRateLimiter(float32(clientQps), clientBurst)
```

Due to controller-runtime specification, IIUC, we need to specify those parameters throughout the RateLimiter.